x-pack/plugins/observability_solution/inventory/common/utils/get_identity_fields_values.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENTITY_IDENTITY_FIELDS } from '@kbn/observability-shared-plugin/common';
+import { Entity } from '../entities';
+
+export function getIdentityFieldValues({ entity }: { entity: Entity }) {
```

Comment:
Do you think we should improve the name of this function? It actually returns the identity fields formated as kql, right?

Reply:
good point, I'm thinking of `getIdentityFieldValuePairToKql` . Let me know if you have a better suggestion 🙏

Reply:
What do you think of `parseIdentityFieldValuesToKql`?

Reply:
done https://github.com/elastic/kibana/pull/195250/commits/b594c5a304e6b14098ecc4c9a2335895e8b86c89